src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -2471,98 +2936,106 @@ await using (var context = InMemoryContentDbContext(contentDbContextId))
-                    File = new File
-                    {
-                        Type = FileType.Data
-                    }
+                    Publication = publicationA,
+                    ReleaseName = "2020",
+                    TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                    Type = ReleaseType.OfficialStatistics,
+                    Published = new DateTime(2020, 1, 1),
                 },
-                new()
+                File = new File { Type = FileType.Data },
+            };
+            var releaseFileB = new ReleaseFile
+            {
+                Release = new()
                 {
-                    Release = amendedRelease,
-                    File = new File
-                    {
-                        Type = FileType.Ancillary
-                    }
+                    Publication = publicationB,
+                    ReleaseName = "2020",
+                    TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                    Type = ReleaseType.NationalStatistics,
+                    Published = new DateTime(2020, 2, 1),
                 },
+                File = new File { Type = FileType.Data },
             };
 
-            var contentDbContextId = Guid.NewGuid().ToString();
-
-            await using (var context = InMemoryContentDbContext(contentDbContextId))
+            var contextId = Guid.NewGuid().ToString();
+            await using (var context = InMemoryContentDbContext(contextId))
             {
-                await context.AddRangeAsync(theme);
-                await context.AddRangeAsync(releaseFiles);
-                await context.AddRangeAsync(amendedRelease);
+                await context.AddRangeAsync(theme, releaseFileA, releaseFileB);
                 await context.SaveChangesAsync();
             }
 
-            await using (var context = InMemoryContentDbContext(contentDbContextId))
+            await using (var context = InMemoryContentDbContext(contextId))
             {
                 var service = BuildThemeService(context);
 
-                var result = await service.GetPublicationTree(PublicationTreeFilter.LatestData);
-                Assert.Empty(result);
+                var result = await service.GetPublicationTree(PublicationTreeFilter.DataCatalogue);
+
+                Assert.Equal(1, result.Count);
+                Assert.Equal("Theme A", result[0].Title);
+
+                Assert.Single(result[0].Topics);
+                Assert.Equal("Topic A", result[0].Topics[0].Title);
+
+                var publications = result[0].Topics[0].Publications;
+                Assert.Equal(2, publications.Count);
+
+                Assert.Equal("Publication A", publications[0].Title);
+                Assert.False(publications[0].IsSuperseded);
+
+                Assert.Equal("Publication B", publications[1].Title);
+                Assert.True(publications[1].IsSuperseded);
             }
         }
 
```

Comment:
Where are all the `FastTrack` tests? :wink: 

Not going to ask you to do these, but you should probably be able to appreciate why having filters for specific screens might not be the best idea.

Reply:
I'm not sure I have an issue with this - the FastTrack tests are covered by the DataCatalogue tests - if they differ in the future, unit tests for FastTrack can be written at that point. I don't see much of a downside.

Reply:
Ideally, we want any tests to be value for money. With the current enum approach, we're having to do a lot of duplication of tests, but is necessary as each enum can represent drastically different scenarios.

It's entirely possible that the conditions for the `FastTracks` enum don't align with the `DataCatalogue` enum in the future and there are no tests to actually catch any problems that might arise from that.

Of course, it's not great to have to do a bunch of duplication to test a different code path, but that's the architecture we've gone with currently. Unfortunately, the flip-side is that leaving the tests to be written by someone in the future isn't very responsible :wink:

We can leave this for now as my money is on us re-writing a lot of this in the near future anyway.